main.tf

```diff
@@ -135,7 +160,7 @@ data "google_compute_default_service_account" "default" {
  *****************************************/
 resource "null_resource" "delete_default_compute_service_account" {
   provisioner "local-exec" {
-    command = "${path.module}/scripts/delete-service-account.sh ${local.project_id} ${var.credentials_path} ${data.google_compute_default_service_account.default.id}"
+    command = "${path.module}/scripts/delete-service-account.sh '${local.project_id}' '${var.credentials_path}' '${data.google_compute_default_service_account.default.id}'"
```

Comment:
Why are we adding quotes around the string variables ?

Reply:
This allows the receiving script to receive positional arguments that are empty, which in turn allows us to tell the difference between scripts that received no arguments, and scripts that were called with an empty `credentials_path` variable. Is this design too clever?

Reply:
This implementation has been replaced, resolving this comment.